vimiv/gui/crop_widget.py

```diff
@@ -0,0 +1,138 @@
+# vim: ft=python fileencoding=utf-8 sw=4 et sts=4
+
+# This file is part of vimiv.
+# Copyright 2017-2020 Christian Karl (karlch) <karlch at protonmail dot com>
+# License: GNU GPL v3, see the "LICENSE" and "AUTHORS" files for details.
+
+"""Widget to display a rectangle for cropping and interact with image and transform."""
+
+from PyQt5.QtCore import Qt, QPoint, QRect, QRectF, QSize
+from PyQt5.QtGui import QPainter
+from PyQt5.QtWidgets import QApplication, QStyle, QStyleOption
+
+from vimiv import api
+from vimiv.config import styles
+
+from .resize import ResizeLayout
+from .transform_widget import TransformWidget
```

Comment:
Maybe this import is the culprit?

Reply:
It definitely is, renaming the import to `transformwidget` seems to run just fine (after the cherry-picks at least)